python_modules/libraries/dagster-duckdb/dagster_duckdb/io_manager.py

```diff
@@ -83,7 +83,7 @@ def my_table_a(my_table: pd.DataFrame):
             ),
         }
     )
-    def duckdb_io_manager(_):
+    def duckdb_io_manager(init_context):
```

Comment:
can you do a check that the api docs build still works with the new parameter? I think it should be fine, but iirc I put the _ in so that we could do the kinda hacky thing that lets the docstrings get picked up by sphinx. If it fails with the new parameter name, lmk and I can help debug, since sphinx is doing some kinda weird stuff here